pkg/ddevapp/mutagen_test.go

```diff
@@ -250,3 +250,48 @@ func TestMutagenConfigChange(t *testing.T) {
 
 	util.Debug("origSyncID=%s afterRestartSyncID=%s afterChangeSyncID=%s", origSyncID, afterRestartSyncID, afterChangeSyncID)
 }
+
+// Test app starts despite a missing mutagen-agents.tar.gz file
```

Comment:
Could this be done at the end of TestMutagenSimple and require less setup that way?

Reply:
Yes, probably lol